nntrainer/layers/layer_node.h

```diff
@@ -255,7 +255,8 @@ class LayerNode : public ml::train::Layer, public GraphNode {
   virtual void
   export_to(Exporter &exporter,
             ExportMethods method = ExportMethods::METHOD_STRINGVECTOR) const {
-    exporter.saveResult(props, method, this);
+    /// waiting for #1272
+    // exporter.saveResult(props, method, this);
```

Comment:
This was causing compile error (props does not exist, so this is commented) can be resolves with #1272

Reply:
please cherry-pick that commit on your local end. Once #1272 is merged to main, we can cherry pick it here too.

Reply:
I'll after #1272 is merged

Reply:
We dicsussed offline to wait for #1284 instead.